octoprint_mrbeam/static/js/app/view-models/modal/laserhead-changed.js

```diff
@@ -5,15 +5,79 @@ $(function () {
 
         self.settings = params[0];
         self.loginState = params[1];
+        self.laserhead_model_id = ko.observable();
+        self.laserhead_model_supported = ko.observable();
+        self.step = ko.observable(1);
```

Comment:
What is step? And what is the difference between it being 1 or 3?

Reply:
The step of the modal windows, for the x head we have 3 different contents showing